frontend/src/queries/schema.ts

```diff
@@ -311,6 +310,18 @@ export interface RecordingsQueryResponse {
     has_next: boolean
 }
 
+export enum RecordingOrder {
+    Duration = 'recording_duration',
+    InactiveSeconds = 'inactive_seconds',
+    ActiveSeconds = 'active_seconds',
+    StartTime = 'start_time',
+    ConsoleErrorCount = 'console_error_count',
+    ClickCount = 'click_count',
+    KeypressCount = 'keypress_count',
+    MouseActivityCount = 'mouse_activity_count',
+    ActivityScore = 'activity_score',
+}
```

Comment:
lots of type errors from this change
i'm not a fan of TS enums
so I'd leave it as moving definition here as

`export type RecordingOrder 'start_time' | 'active_seconds' | 'etc'`

Reply:
> i'm not a fan of TS enums

why? ...

* it's not always straight-forward to know if you're dealing with the key, the value, or the implied numeric index
* they almost always add text to read and duplicate content `StartTime = 'start_time'` doesn't add much for me
* if you need to loop the values you can do the `array as const` trick

I'm also not a believer that this kind of opinion has a lot of value so feel free to fix the type errors rather than changing the type declaration

Reply:
I personally like enums BUT I think not using an enum will make it easier to fix the type errors, so I will go with your suggestion 😄